test/e2e/e2e_suite_test.go

```diff
@@ -47,11 +48,34 @@ const (
 	VsphereStoragePolicy = "VSPHERE_STORAGE_POLICY"
 )
 
+const (
+	// GovmomiTestMode identify tests run with CAPV using govmomi to access vCenter.
+	GovmomiTestMode string = "govmomi"
+
+	// SupervisorTestMode identify tests run with CAPV in supervisor mode (delegating to vm-operator all the interaction with vCenter).
+	SupervisorTestMode string = "supervisor"
+)
+
+const (
+	// VMCTestTarget identify tests targeting VMC infrastructure used for CAPV CI.
+	VMCTestTarget string = "vmc"
+
+	// VCenterTestTarget identify tests targeting a user provided vCenter instance.
+	VCenterTestTarget string = "vcenter"
+
```

Comment:
This one seems not to be in use yet? What is supposed to be different for VMC and VCenter?